platform/metrics/platform-metrics-collector/src/main/java/ddf/metrics/collector/rrd4j/RrdJmxCollector.java

```diff
@@ -353,7 +353,7 @@ private boolean isMbeanAccessible() {
           LOGGER.trace("MBean [{}] not found, sleeping...", mbeanName);
           Thread.sleep(1000);
         } catch (InterruptedException ie) {
-          // Ignore this
+          Thread.currentThread().interrupt();
```

Comment:
❗️ Flag the interruption and propagate outside of while loop in a finally clause.